src/confd/yang/infix-services.yang

```diff
@@ -36,6 +59,49 @@ module infix-services {
       type boolean;
     }
   }
+  container ssh {
+    description "Configuration for the SSH daemon";
+    leaf enabled {
+      description "Disable or enable SSH daemon";
+      type boolean;
+      must ". = 'false' or (count(../listen) > 0 and count(../hostkey) > 0)" {
+        error-message "When SSH is enabled, there must be at least one listen address configured and a private key must be set.";
+      }
+    }
+
+    leaf-list hostkey {
+      description
+        "A reference to an asymmetric key that exists in
+          the central keystore.";
+      must "not(deref(.)/../ks:public-key-format) or "
+        + "(derived-from-or-self(deref(.)/../ks:public-key-format,  'ct:ssh-public-key-format') and"
+        + "derived-from-or-self(deref(.)/../ks:private-key-format,  'ct:rsa-private-key-format'))" {
+        error-message "Only RSA hostkeys are supported";
+      }
+      type ks:asymmetric-key-ref;
+    }
+
+    list listen {
+      key name;
+      leaf name {
+        type string;
+      }
+
+      leaf address {
+        description
+          "The local IP address to listen on for incoming
+           SSH client connections.  INADDR_ANY (0.0.0.0) or
```

Comment:
Same here, make it easier for users of the CLI help system.  Suggestion to simplify first sentence and then break up the rest in a separate paragraph with emtpy line between. (See above)

    description "Local IP address to listen on for inbound SSH connections.

                 INADDR_ANY ... "